db/schema.rb

```diff
@@ -37,6 +49,15 @@
     t.index ["user_id", "user_type"], name: "index_ahoy_messages_on_user_id_and_user_type"
   end
 
+  create_table "ahoy_visits", force: :cascade do |t|
+    t.datetime "started_at"
+    t.bigint "user_id"
+    t.string "visit_token"
```

Comment:
Are these tokens just something at Ahoy uses to track users for its own purposes?

Reply:
@mstruve Good question! The ahoy docs don't really explain it at all (_le sigh 😞_) but as far as I can tell from looking at the source code, `visit_token` is a unique uuid identifier to create/identify/track visits. It is a required column, and the gem internally relies on its uniqueness.